packages/testing/src/index.js

```diff
@@ -6,4 +6,5 @@
  */
 
 export { default as mountWithTheme } from './utils/mountWithTheme.js';
+export { default as renderWithTheme } from './utils/renderWithTheme.js';
```

Comment:
I'm a little lost as well. The only difference I can see is that `mountWithTheme` is passing the childContext types through the context with `{ context, childContextTypes: StyledThemeProvider.childContextTypes },`

Without that you are probably seeing some Styled-Components `console.warn`s in the console?

Reply:
@ryanseddon the only difference from the `mountWithTheme` is that it calls `render` instead of `mount` for the passed component. And this is the main purpose of this helper.

@austin94 good catch! Added `childContextTypes` to the options.

Reply:
What use-case would this have for testing? The `react-testing` packages are mostly around utilities for enzyme & jest testing.

Reply:
This is exactly for snapshots testing with enzyme. Useful when a component composites smaller none exportable components and mounting to DOM is not needed. `shallow` doesn't render subcomponents and `render` is doing this down to html elements.

Reply:
Thanks for the clarification @ssidorchik. Is this change intended to clean up some of our tests or are you depending on this in your own codebase for testing?

Reply:
Already using the enzyme's `render` on a separate project. And would like to add the theme support.